x-pack/plugins/enterprise_search/server/routes/enterprise_search/indices.ts

```diff
@@ -141,15 +145,21 @@ export function registerIndexRoutes({ router, log }: RouteDependencies) {
       const { client } = (await context.core).elasticsearch;
 
       try {
-        const connector = await fetchConnectorByIndexName(client, indexName);
         const crawler = await fetchCrawlerByIndexName(client, indexName);
+        const connector = await fetchConnectorByIndexName(client, indexName);
 
-        if (connector) {
-          await deleteConnectorById(client, connector.id);
+        if (crawler) {
+          const crawlerRes = await enterpriseSearchRequestHandler.createRequest({
```

Comment:
question: will ent-search also try to delete the connector record, or leave that alone?

Reply:
@sphilipse ent-search won't delete connector records, as they're being deleted here anyway.